src/Store/StoreThread.re

```diff
@@ -15,6 +15,8 @@ module Model = Oni_Model;
 
 open Oni_Extensions;
 
+module Log = (val Core.Log.withNamespace("StoreTread"));
```

Comment:
We could also use a functor for this. In that case it would look like:

```reason
module Log = Core.Log.WithNamespace({ let namespace = "StoreThread" });
```

None of them are particularly intuitive, but using `val` is at least less noisy.

Reply:
Thanks for calling out the possibilities! I agree, the `val` first-class module is at least more concise